docs/addons/writing-addons.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Writing your own Storybook addon'
+---
+
+One of Storybook's main features it's is robust addon ecosystem. Addons can be used to enhance your developer experience or workflows. We're going to guide you with the necessary steps to create your own first addon.
```

Comment:
`is it's`

Reply:
Thanks for catching that one. Updated with the latest commit